backend/test/models/project_riff_test.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+# Tests for ProjectRiffs
+class ProjectRiffTest < ActiveSupport::TestCase
+  #   Assert that a ProjectRiff without a project_id is invalid.
+  test 'a ProjectRiff without a project_id is invalid' do
+    project_riff = build(:project_riff)
+    assert_not project_riff.valid?
+    assert_includes project_riff.errors[:project_id], "can't be blank"
+  end
+
+  #   Assert that a ProjectRiff without a riff_id is invalid.
+  test 'a ProjectRiff without a riff_id is invalid' do
+    project_riff = build(:project_riff)
+    assert_not project_riff.valid?
+    assert_includes project_riff.errors[:riff_id], "can't be blank"
+  end
+
+  #   Assert that a riff_id must be unique per project_id (scoped uniqueness).
```

Comment:
Should we also test that different riffs in the same project is valid and that the same riff in different projects is valid?